source/features/forked-to.tsx

```diff
@@ -12,10 +12,10 @@ import LinkExternalIcon from 'octicon/link-external.svg';
 import features from '.';
 import fetchDom from '../helpers/fetch-dom';
 import GitHubURL from '../github-helpers/github-url';
-import {getRepoURL, getUsername, getForkedRepo} from '../github-helpers';
+import {getUsername, getForkedRepo, getRepositoryInfo} from '../github-helpers';
 
-const getForkSourceRepo = (): string => getForkedRepo() ?? getRepoURL();
-const getCacheKey = (): string => `forked-to:${getUsername()}@${getForkSourceRepo().toLowerCase()}`;
+const getForkSourceRepo = (): string => getForkedRepo() ?? getRepositoryInfo()!.url;
+const getCacheKey = (): string => `forked-to:${getUsername()}@${getForkSourceRepo()}`;
```

Comment:
This feature was one of the affected by the capitalization:

- `getForkedRepo` already reported the canonical capitalization
- `getForkSourceRepo` now _always_ reports the canonical capitalization, so it doesn't need to be lowercased anywhere (it wasn't necessary before either since `getRepoURL` was lowercased already)